store/types/store.go

```diff
@@ -322,6 +324,9 @@ func (st StoreType) String() string {
 
 	case StoreTypePersistent:
 		return "StoreTypePersistent"
+
+	case StoreTypeSnapshot:
```

Comment:
wait, why a snapshot a store? It doesn't persist anything nor deal with R/W capabilities.

Reply:
At first, I was confused as to why the snapshot store does not implement the `Store` interface so I attempted to make it so. However, then, I realized that snapshot store is different from the rest and reverted that change while forgetting to revert this piece of code.

I removed this now. Thanks for catching it.

Just a note, the snapshot store does persist snapshots to the db: https://github.com/osmosis-labs/cosmos-sdk/blob/70c2043a021456f435a64571ad9c1b9a14a40305/snapshots/store.go#L302 but this abstraction is unrelated to the `Store` interface